Sources/Cocoa/Components/CarouselView/CarouselCollectionView.swift

```diff
@@ -252,6 +256,10 @@ final class CarouselCollectionView
     func setCurrentIndex(_ index: Int, animated: Bool = true, completion: (() -> Void)? = nil) {
         guard currentIndex != index else { return }
 
+        if style == .default, index > (numberOfItems - 1) || index < 0 {
```

Comment:
Fix the `.default` style autoscrolling out of the items when infinite scrolling is dissabled.